plugins/chat/app/queries/chat/tracking_state_report_query.rb

```diff
@@ -59,24 +60,40 @@ def self.call(
       if !include_threads || (thread_ids.blank? && channel_ids.blank?)
         report.thread_tracking = {}
       else
+        tracking =
+          ::Chat::ThreadUnreadsQuery.call(
+            channel_ids: channel_ids,
+            thread_ids: thread_ids,
+            user_id: guardian.user.id,
+            include_missing_memberships: include_missing_memberships,
+            include_read: include_read,
+          )
+
+        last_reply_details =
```

Comment:
This is not super ideal, but better than trying to contort the ThreadUnreadsQuery to do it, since the latter doesn't really need the data.